posix/basic-unixint.lisp

```diff
@@ -298,6 +298,12 @@
   (ctype blksize "long")
   (ctype blkcnt "long"))
 
+#+darwin
+(cstruct timespec "struct timespec"
```

Comment:
Export struct name and fields. Remove the conditional: struct timespec is defined by POSIX.